src/main/fc/cli.c

```diff
@@ -402,12 +402,18 @@ static uint16_t getValueOffset(const clivalue_t *value)
     return 0;
 }
 
-STATIC_UNIT_TESTED void *getValuePointer(const clivalue_t *value)
+void *cliGetValuePointer(const clivalue_t *value)
 {
     const pgRegistry_t* rec = pgFind(value->pgn);
     return CONST_CAST(void *, rec->address + getValueOffset(value));
 }
 
+const void *cliGetDefaultPointer(const clivalue_t *value)
```

Comment:
Any reason for this function?

Reply:
Yes, it's to support some work I'm doing on the CRSF protocol.